content/articles/nesting-navigators-and-routing-with-react-navigation-in-react-native/index.md

```diff
@@ -0,0 +1,435 @@
+
+When you nest navigators, the screens of one navigator are rendered inside the screens of another. If there is a stack, switching to a different screen will cause a new screen to be displayed.
+
+Navigators are in charge of making the switch between different screens. There are numerous navigation types supported by React Navigation, including Stack, Drawer, and Tab Navigators. In addition to navigating between screens, it can be used to transfer information between them.
+
+Table of content:
+- [First Step- Application development and module installation](#first-step--application-development-and-module-installation)
+- [Second Step- Creating a HomeScreen and NewworkerScreen](#second-step--creating-a-homescreen-and-newworkerscreen)
+- [Third Step - Navigating React Pages using StackNavigator](#third-step--navigating-react-pages-using-stacknavigator)
+- [Fourth Step - Adding Buttons to HomeScreen and NewWorkerScreen](#fourth-step--adding-buttons-to-homescreen-and-newworkerscreen)
+- [Fifth Step- Passing Data to Other Screens Using Context](#fifth-step--passing-data-to-other-screens-using-context)
+
+### Prerequisites
+You'll require the following supplies to follow along with this tutorial:
+
+1. An on-disk Node.js development environment. Make a Local Development Environment with Node.js and the [Installation Guide.](https://www.digitalocean.com/community/tutorial_series/how-to-install-node-js-and-create-a-local-development-environment)  
+
+2. An advantage for this project is familiarity with the iOS or Android simulators and setting up a new React Native environment in your development environment.
+
+### Goals of the Tutorial
+1. This post will teach you how to make a multi-screen React Native app.
+2. React Navigation will teach you how to navigate from one screen to the next if you're working with the library.
+
+There will be several steps that we will walk through to achieve our goal in this tutorial.
+
+### First Step- Application development and module installation
+
+1. The next step is to use the commands below to create a new project.
+
+```bash
+$ npx react-native init react-navigation-routing --version 0.63.2
+```
+
+2. After that, go to the newly created directory by using the following command here:
+
+```bash
+$ cd react-navigation-routing
+```
+
+3. Use the following command to install the necessary packages:
+
+```bash
+npm install –save react-navigation react-navigation-stack react-native-reanimated react-native-gesture-handler react-native-screens react-native-vector-icons
+```
+
+4. Open the iOS or Android application, depending on your device respectively using the following code:
+
+```bash
+$ npm run ios
+
+$ npm run android
+```
+
+### Second Step- Creating a HomeScreen and NewWorkerScreen

```

Comment:
What do you mean by NewWorkerScreen? There's no context for this. Explain what you are trying to achieve with the app before starting on the screens.